manifests/init.pp

```diff
@@ -338,6 +338,7 @@
       postgresql_db_ssl_cert    => $pulpcore_postgresql_ssl_cert,
       postgresql_db_ssl_key     => $pulpcore_postgresql_ssl_key,
       postgresql_db_ssl_root_ca => $pulpcore_postgresql_ssl_root_ca,
+      before                    => Class['foreman_proxy::plugin::pulp'],
```

Comment:
Could we do it before the register?

```
iff --git a/manifests/init.pp b/manifests/init.pp
index 9bea8db..78d833a 100644
--- a/manifests/init.pp
+++ b/manifests/init.pp
@@ -338,6 +338,7 @@ class foreman_proxy_content (
       postgresql_db_ssl_cert    => $pulpcore_postgresql_ssl_cert,
       postgresql_db_ssl_key     => $pulpcore_postgresql_ssl_key,
       postgresql_db_ssl_root_ca => $pulpcore_postgresql_ssl_root_ca,
+      before                    => Class['foreman_proxy::register'],
     }
 
     if $pulp_master {
```

Reply:
Tried that, same error.

Reply:
Strange, I made this change, ran tests. Everything passed. I ran it again, got these failures. Perhaps my fixtures weren't up to date? Implies it was a more recent change that led to this?

Reply:
Yes, https://github.com/theforeman/puppet-puppet/commit/4302973e8f47c77ed35b2a399c1ccef2d503b400